2.0-PROPOSAL/README.md

```diff
@@ -483,6 +494,11 @@ N.B.: In the case of projects, JSON schema can be reused by IDEs to validate
 project definitions as they are edited. The author proposes this UX to be
 superior to the current `brig project create` process.
 
+As with Kubernetes, Brigade 2.0's API surface is likely to be logically
+decomposed into several smaller APIs foucsed on specific  areas of concern to
```

Comment:
> foucsed on specific  areas of concern
focused on specific areas of concern

Reply:
Fixed in 7dade9ed162f73f3659ad20c34844893a616da8f